src/test/java/pages/HomePage.java

```diff
@@ -1,12 +1,21 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class HomePage extends BasePage {
 
+
     By userAvatarIcon = By.cssSelector("img.avatar");
+    static By playlistLocator = By.cssSelector(".playlist:nth-child(3)");
+    static By playlistNameLocator = By.cssSelector("input[name='name']");
+
+    static By playlistElementLocator = By.xpath("//a[contains (text(), 'name')]");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");